src/EditorFeatures/Core.Wpf/InlineRename/Dashboard/DashboardAdornmentManager.cs

```diff
@@ -60,6 +62,8 @@ private void UpdateAdornments()
             if (_renameService.ActiveSession != null &&
                 ViewIncludesBufferFromWorkspace(_textView, _renameService.ActiveSession.Workspace))
             {
+                _dashboardColorUpdater?.UpdateColors();
+
```

Comment:
Does this work with theme updates after opening?

Reply:
Yes, since the thing we're passing around here is the keys, not the colors/brushes themselves. Turns out that's what makes the other approaches difficult -- it's not just the values that are dynamic, but the keys themselves.